### DISCUSSION TITLE
trying to add custom DNS entry in keda

### DISCUSSION THREAD

**Discussion by TechnicalShivam** [kedacore/keda#5265] [Q&A / Need Help]
apiVersion: keda.sh/v1alpha1
kind: ScaledObject
metadata:
  name: kpit-github-runner-scaledobject
  namespace: keda
spec:
  scaleTargetRef:
    name: generic-workflow-2
  minReplicaCount: 1
  maxReplicaCount: 10
  triggers:
    - type: github
      metadata:
        githubAPIURL: "private-dns"
        owner: "blank-project"
        runnerScope: "ent"
        repos: "k3s"
        labels: "sudo-runner-dns"
        targetWorkflowQueueLength: "1"
      authenticationRef:
        name: github-trigger-auth
        
  Warning  KEDAScalerFailed    46m (x6 over 49m)     keda-operator  the GitHub REST API returned error. url: https://api.github.com/repos/blank-project/k3s/actions/runs status: 401 response: {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest"}
  
 we are using our personal github personal account that contain different domain i.e. https://api.githubtls.bis.com but by default it is going to api.github.com only. Is KEDA supports private github account or only support public github?

**Comment by JorTurFer** [comment]
This looks like an error, let's continue in the issue
